tests/integration/test_cluster.py

```diff
@@ -0,0 +1,240 @@
+from pathlib import Path
+from typing import Collection
+from urllib.parse import urljoin
+import asyncio
+
+from httpx import AsyncClient
+import pytest
+import pytest_asyncio
+
+from adcm_aio_client.core.client import ADCMClient
+from adcm_aio_client.core.config import ConfigHistoryNode, ObjectConfig
+from adcm_aio_client.core.errors import MultipleObjectsReturnedError, ObjectDoesNotExistError
+from adcm_aio_client.core.mapping import ClusterMapping
+from adcm_aio_client.core.objects.cm import Bundle, Cluster
+from tests.integration.bundle import pack_bundle
+from tests.integration.conftest import BUNDLES
+
+pytestmark = [pytest.mark.asyncio]
+
+
+def _get_httpx_client_base_url(adcm_client: ADCMClient) -> tuple[AsyncClient, str]:
+    httpx_client = adcm_client._requester._client  # pyright: ignore[reportAttributeAccessIssue]
+    base_url = urljoin(str(adcm_client._requester._client.base_url), "api/v2/")  # pyright: ignore[reportAttributeAccessIssue]
```

Comment:
can you please elaborate on this requester using approach: motivation, profits, etc.?

Reply:
it is used for checking responses from ADCM / updating entities to test `refresh()` behavior. Just got existing client from fixtures. Are you suggesting creating new httpx_client for this?

Reply:
replaced with `conftest.httpx_client` fixture